api/services/handle_event_service.py

```diff
@@ -6,16 +6,15 @@
 
 
 class HandleEventService:
-    def handle_event_service(self, request):
-
+    def handle_event_service(self, form_data, log_created_on=None):
         call_sid_exist = db.session.query(
-            db.exists().where(models.CallLogEvent.call_sid == request.form["CallSid"])

```

Comment:
previously we are passing the whole request object but as we have the paylaod as json so to be in sync we are now passing `request.form` for regular webhooks and jsonData for retry webhooks. 

This way both the data can be handle using the same logic